src/stores/photo.js

```diff
@@ -10,18 +10,49 @@
   See the License for the specific language governing permissions and
```

Comment:
Database and table assertion. Like that this is broken into multiple functions. Would potentially benefit from use of `async` and `await`, but not substantially enough to bother with.